### PR TITLE
RMMIS-6466: Update Page, ES6ify, render-children module, update unit …

### PR DESCRIPTION
**Pull request by traviscooperscdhhs**
…tests

### CODE REVIEW

**Comment by kjschulz** [comment]
:+1:

File: src/Page.jsx [hunk 1]
```diff
@@ -1,36 +1,35 @@
 'use-strict';
 var React = require('react');
```
Comment (mallenscdhhs) [inline src/Page.jsx]: ES6-ify these jokers

Comment (traviscooperscdhhs) [inline src/Page.jsx]: (dadgumit) (facepalm)

Comment (traviscooperscdhhs) [inline src/Page.jsx]: (whyyouno) see the VAR (travis)!

**Pull request merged**